mparray/special/_special.py

```diff
@@ -90,12 +99,15 @@ def xlog1py(x, y):  # needs accuracy review
 
 @vectorize
 def cosm1(x):
+    if x == 0:
+        return mp.zero
```

Comment:
Just a shortcut? It seems to give the right result without this.

Reply:
Yeah. I think this was just done mindlessly out of habit

Reply:
```suggestion
```